CppSamples/DisplayInformation/ShowGrid/ShowGrid.qml

```diff
@@ -19,24 +24,116 @@ import QtQuick.Controls
 import QtQuick.Layouts
 import Esri.Samples
 
-DisplayGridSample {
-    id: displayGrid
-    clip: true
-    width: 800
-    height: 600
-
-    property real centerWindowY: (displayGrid.height / 2) - (styleWindow.height / 2)
+Item {
```

Comment:
❓ Typically, the *Sample is the root type. Is the Item being root necessary?

Reply:
I'm not sure what you mean? Here's our most recent template: https://github.com/Esri/arcgis-maps-sdk-samples-qt/blob/v.next/Templates/CppSampleTemplate/sample.qml.tmpl#L21